packages/odo-expandable/src/expandable.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * @fileoverview An basic, expandable component that has both a trigger
+ * and a target to open.
+ *
+ * @author Matt Zaso
+ */
+import Settings from './expandable-settings';
+import ExpandableGroup from './expandable-group';
+
+class Expandable {
+  constructor(id, options = {}) {
+    /** @type {string} */
+    this.id = id;
+
+    /**
+     * Override any defaults with the given options.
+     * @type {Object}
+     */
+    this.options = Object.assign({}, Settings.Defaults, options);
+
+    /** @type {Element} */
+    this._trigger = document.body.querySelector(`[${Settings.Attribute.TRIGGER}="${id}"]`);
+
+    /** @type {Element} */
+    this._target = document.body.querySelector(`[${Settings.Attribute.TARGET}="${id}"]`);
+
+    this._setARIAAttributes();
+
+    this._bindListeners();
+  }
+
+  /**
+   * Binds the listeners to the body to handle click events.
+   * @private
+   */
+  _bindListeners() {
+    if (!this.options.groupedItem) {
+      this._onTriggerClick = this._triggerClickHandler.bind(this);
+      document.body.addEventListener('click', this._onTriggerClick);
+    }
+  }
+
+  /**
+   * Handler for clicks on the trigger.
+   * @param {Event} evt Event object.
+   * @private
+   */
+  _triggerClickHandler(evt) {
+    evt.preventDefault();
+    const closest = evt.target.closest(`[${Settings.Attribute.TRIGGER}]`);
+
+    if (closest !== null && closest === this._trigger) {
+      this.toggle();
+    }
+  }
+
+  /**
+   * Sets the appropriate ARIA attributes for a11y.
+   * @private
+   */
+  _setARIAAttributes() {
+    const elementId = `expandable-${this.id}`;
+
+    this._trigger.setAttribute('aria-describedby', elementId);
+
```

Comment:
I think you need a couple aria attributes.
![screenshot 2018-02-09 14 09 01](https://user-images.githubusercontent.com/963921/36052510-d6eef8c0-0da2-11e8-8461-1d042d431386.png)